cli/tests/no_mem_cache.js

```diff
@@ -0,0 +1,33 @@
+const fixtureFile = await Deno.makeTempFile();
+let prefix = "file://";
+if (Deno.build.os == "windows") {
+  prefix += "/";
+}
```

Comment:
Aside: I wish had something like `Deno.makeTempFile().toUrl()`

Reply:
@ry IMO this should be handled by `URL`. This bit is very problematic in other contexts, especially in dynamic imports: https://github.com/denoland/deno/issues/4641

Unfortunately JS `URL` can't resolve it properly just like `Url` Rust crate 😢